c2corg_ui/templates/waypoint/helpers/detailed_waypoint_attributes.html

```diff
@@ -0,0 +1,365 @@
+<%namespace file="../../helpers/view.html" import="show_areas, show_attr"/>
+
+## MAPSINFO
+<%def name="get_waypoint_maps_info(waypoint)">\
+% if waypoint.get('maps_info'):
+<div class="name-icon-value" ng-click="detailsCtrl.openTab($event)">
+  <h4><span translate>maps_info</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+  <div class="name-icon">
+    <span class="glyphicon glyphicon-map-marker"></span>
+    <p translate>maps_info</p>
+  </div>
+  <span class="detail-text accordion">
+    <p><span translate>maps_info</span>: <b>${waypoint['maps_info']}</b></p>
+  </span>
+</div>
+% endif
+</%def>
+
+## GENERAL
+<%def name="get_waypoint_general(waypoint)">\
+<div class="name-icon-value general" ng-click="detailsCtrl.openTab($event)">
+  <h4><span translate>General</span><span class="glyphicon glyphicon-menu-right"></span></h4>
+  <div class="name-icon">
+    <span class="glyphicon glyphicon-asterisk"></span>
+    <p translate>General</p>
+  </div>
+  <span class="detail-text accordion">
+    % if waypoint.get('waypoint_type'):
+    <p><span translate>waypoint_type</span>: <b x-translate>${waypoint['waypoint_type']}</b></p>
+    % endif
+
+    % if waypoint.get('orientation'):
+      <p><span translate>orientation</span>: <b>${waypoint['orientation']}</b></p>
```

Comment:
I think we need an ``x-translate`` for ``${waypoint['orientation']}`` since the shorts for cardinal points are different from a language to another, especially E and W.

By the way (but it's a nice to have), it would be cool to show orientations graphically, using a compass.